UM/Settings/ContainerStack.py

```diff
@@ -314,7 +314,7 @@ def getVersionFromSerialized(self, serialized: str) -> Optional[int]:
     #   Reimplemented from ContainerInterface
     #
     #   TODO: Expand documentation here, include the fact that this should _not_ include all containers
-    def deserialize(self, serialized):
+    def deserialize(self, serialized, connect_signals=True):
```

Comment:
This adds a boolean trap. At the very least make it `def deserialize(self, serialized, *, connect_signals = True)` since that would force it to always be a keyword argument. However, it also breaks with the ContainerInterface interface.